google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LogEntry.java

```diff
@@ -505,6 +529,7 @@ private static Long millisFromTimestamp(Timestamp timestamp) {
     if (spanId != null) {
       builder.setSpanId(spanId);
     }
+    builder.setTraceSampled(traceSampled);
```

Comment:
Do we need to null check this?

Reply:
The protocol buffer LogEntry.Builder's traceSampled field is a boolean ([link](https://github.com/googleapis/google-cloud-java/blob/f69e7fafe748bc8ce49c2a8d9375b3b3a4644d5d/google-api-grpc/proto-google-cloud-logging-v2/src/main/java/com/google/logging/v2/LogEntry.java#L4141)), so I thought that it wouldn't be necessary to distinguish between false and not set.  The traceSampled field in this class is also a boolean.

Reply:
Gah you're right. Sorry I have been working with Apiary for too long!